### PR TITLE
7 vote on a review

### PR DESCRIPTION
**Pull request by kontsio1**

### CODE REVIEW

File: src/components/DetailedReviewCard.jsx [hunk 1]
```diff
@@ -1,35 +1,64 @@
...
...bzQpLlOaQVrsfhTbTCNpBwnDPIwqqkJs5a1Owb5HgdpaNkWYJBNRwt6wyClHQtelj0GTXSiV7BWFaCVSroonBrxioM01aTMdb6DdXgMsh49bDUGuc9xEIomg577zXFnUO56/xltWmhAnGaQvaiDTukM0iGhhG8cpMP+UC/3n200t4AzBtl01kSlZV3uPxs6a4KqE72CFReS0w/QYHM6eYHXKmZP66BTuophVLBuyQ58jkGY/oyi03pPvXnL7iEG6SUO0gmD8LOsMwZZKj0j4YacWx9u4LU+g401KZ1JRvwZ4+A1ygwrk1lf5rimltvpxK5g3eZowT2Jg9Tesdg1DJLHQVrff6bjWwmrDxItHtqJOYMc2slrN/ehU0+nJU1yLt3NZfizTWOQpWjBIBhkyGjBfcUgm4+DbHH1SJ5HJpEEDKlB2gW3lSdFk2YZZQx/VusOVp5r6vUrz5U7WPv21N/zQbpbxucOVnVb4iAN7mDFV5BkF6qIBrm8hHqNiZRdY2JK2TUmvOIgOtGJpglTJzx3JOZI24pB+qxE2C5P/T0OsnsYJI2DdMIgyX+77Fh8z5poaBm33IKdsKwVBrkH1SMSsUwiUiTSZphLyrxUYs8w6OpxalmlTrQ2WzofpKp6ZCszN6urR7BN37Mm7lscZNO4MakeKfeeZriRzCh5tWjd68kTebzijUxNalyZnsyK3StYv1ePNLRs49EC2XvL5RyevaiGTlwxCDDuxlb/NN4D0wY6sStY11+eyIrymfrZDzxo33TqXOhY0KRQSVddwVoDg8RAioDsYZLxSU2EizJOTriIwaUv4DdlwuFnhsjCIGydyKzfDwYhza/UdfeEyw8/KColEZ02ikF2xLIWFayrltURohIhtONYknghEdJSJ4HsKpqU/UWNsee5dFGhyeDqbqHqBc5YL6SuTp3jICRClHp+EpfRm4gTi/ChQ3CnPwyS+g5p0YfKQkwipK6TVcjg4OWpt9EyztUjiwrWgmVKUalcyJrgTAE+KC0towjBjChQUpbqlp7Rsp3aVY31IqQkrnP1SGnWLzRcnaM49LVCNlXBuiOe+n5hkD4cSHHbQ2wiRF8VIheFtNJJkvPbi6quUazgWK82EsdDyHcMsg2e+v5ikKaWtRr3lMmjS0fqRUi5N3adQZpMA4MK+T+FyojdibNCPQAAAABJRU5ErkJggg=='
```
Comment (Dutton1990) [inline src/components/DetailedReviewCard.jsx:7]: If you want to hide this you could put it in a separate file and require it in 😄

Comment (kontsio1) [reply src/components/DetailedReviewCard.jsx:7]: oh XD

Comment (kontsio1) [reply src/components/DetailedReviewCard.jsx:7]: It was on a single line on mine so I didn't notice it

**Comment by Dutton1990** [comment]
Nice! You can merge this ticket well done!

**Pull request merged**